7.redundancy_analysis/notebooks/0.redundancy_analysis.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # shuffle the data both rows and columns
</pre><p>Does this also shuffle the columns? It looks like it shuffles the row values for each column?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Good catch:</p><p>Change in documentation:</p><p># shuffle the data for each column</p><p># this changes the single well values but keeps the distribution of each column</p>